Implement Camelcase Filter in nunjucks

yjaeseok opened the pull request:
We cannot use camelCase method name before
So I add Camelcase Filter in nunjucks
After this patch you can use camelcase filter in nunjucks file

e.g. (template/interface_header.njk:Line 35)

```Napi::Value {{member.name | camelcase-}}(const Napi::CallbackInfo& info);```

you can use camelcase filter instead of uppercase

ISSUE=#82

yjaeseok committed:
Implement Camelcase Filter in nunjucks

We cannot use camelCase method name before
So I add Camelcase Filter in nunjucks
After this patch you can use camelcase filter in nunjucks file

ISSUE=#82

romandev commented on examples/calculator.cc:
```diff
@@ -22,6 +22,10 @@ void Calculator::Print(const std::string& message) {
   printf("%s\n", message.c_str());
 }
 
+void Calculator::CamelCase() {
```
Could you please rename Equals() method to IsEquals() without adding this new method?
(We wanted to use IsEuqals() but used Equals() due to this camel case problem.)

yjaeseok replied:
Okay I will do this

romandev reviewed (approved):
lgtm with nit

yjaeseok committed:
Remove CamelCase function

Remove CamelCase function because we do not need this function.
And Update equals function name to isEquals.
The function is for showing CamelCase correct apply.

yjaeseok commented:
@romandev as you mentions, I update PR.

Remove CamelCase function and Update equals function name to isEquals.
Please re-review and approve. Thanks :+1:
